axiom/nr_segment_datastore.c

```diff
@@ -24,7 +54,6 @@ static char* create_metrics(nr_segment_t* segment,
   char* rollup_metric = NULL;
   char* scoped_metric = NULL;
   char* statement_metric = NULL;
-  char* instance_metric = NULL;
```

Comment:
I don't see how to add a comment for existing code - but on the line above this (line 52 when writing this)
`nrtxn_t* txn = segment->txn;`
I believe it would be important to add a `NULL` check for `segment`.

Reply:
`segment` is checked for `NULL` by the caller.